config/routes.rb

```diff
@@ -9,9 +9,8 @@
 end
 
 scope 'ticket' do
-  post 'ticket', to: 'ticket#create', as: :buy
-
-  post 'ticket', to: 'ticket#booked'
+  post "ticket", to: 'ticket#create', as: 'ticket_create'
+  post "ticket", to: 'ticket#bookin', as: 'ticket_bookin'
 
```

Comment:
In creating standard REST API we very rarely changing name of methods used in controllers. Please consider how to make API consistent by using only `index/create/update/delete` methods.